src_blog/_posts/New-Release-Grain-v0.4.0-Cereal.md

```diff
@@ -0,0 +1,67 @@
+---
+title: New Release! Grain v0.4.0 - Cereal
+subtitle: Announcing the release of Grain v0.4.0, Cereal, with NEAR support.
+date: 2021-09-07 12:00:00
+author: Blaine Bublitz
+cover: /blog/photos/cover/marie-bellando-mitjans-cBChXxQqEKM-unsplash.jpg
+coverAttribution: Marie Bellando-Mitjans / Unsplash
+tags: Release, Changelog
+---
+
+After nearly 5 months of hard work, we are excited to finally share Grain v0.4.0 with the world! This release marks the culmination of a bunch of parallel features being built by the core team and a ton of amazing contributions from our community. 🎉
```

Comment:
```suggestion
After nearly 5 months of hard work, we are excited to finally share Grain v0.4.0 with the world! This release marks the culmination of a bunch of exciting features being built by the core team and a ton of amazing contributions from our community. 🎉
```
style nit (didn't feel like "parallel" was a good word here)

Reply:
```suggestion
After nearly 5 months of hard work, we are happy to finally share Grain v0.4 with the world! This release marks the culmination of a bunch of exciting features being built by the core team and a ton of amazing contributions from our community. 🎉
```

Reply:
didn't want to say exciting and excited in the same paragraph